packages/client/screens/UserProfile/UserFollowersSearch/__tests__/UserFollowersSearch.test.tsx

```diff
@@ -0,0 +1,719 @@
+                () => ({}),
+            'Load more followers results': () => ({}),
+            'fill searched user nickname query and submit': () => ({}),
+            'fill my nickname and submit': () => ({}),
+            'cancel search bar': () => ({}),
+            'press an other user card': () => ({}),
+            'press my user card': () => ({}),
+        },
+    },
+);
+
+const searchUserFollowerMachine = searchUserFollowerModel.createMachine({
+    id: 'User followers search engine',
+    initial: 'Initializing',
+    states: {
+        Initializing: {
+            meta: {
+                test: ({ screen }: TestingContext) => {
+                    expect(screen).toBeUndefined();
+                },
+            },
+            on: {
+                'Make API respond user not found and render application': {
+                    target: '#User followers search engine.User not found',
+                },
+                'Make API respond user found and render application': {
+                    target: '#User followers search engine.User found.User relations are viewable',
+                },
+                'Make API respond forbidden exception and render application': {
+                    target: '#User followers search engine.User found.User relations are forbidden',
+                },
+            },
+        },
+
+        'User not found': {
+            meta: {
+                test: async ({ screen }: TestingContext) => {
+                    invariant(
+                        screen !== undefined,
+                        'Screen must have been rendered before being in this state',
+                    );
+
+                    await waitFor(() => {
+                        const userFollowersSearchContainer = screen.getByTestId(
+                            `search-user-followers-screen`,
+                        );
+                        expect(userFollowersSearchContainer).toBeTruthy();
+                        const userNotFoundText = within(
+                            userFollowersSearchContainer,
+                        ).getByText(/User.*not.*found/i);
+                        expect(userNotFoundText).toBeTruthy();
+                    });
+                },
+            },
+        },
+
+        'User found': {
+            meta: {
+                test: async ({ screen }: TestingContext) => {
+                    invariant(
+                        screen !== undefined,
+                        'Screen must have been rendered before being in this state',
+                    );
+
+                    await waitFor(() => {
+                        const userFollowersSearchContainer = screen.getByTestId(
+                            `search-user-followers-screen`,
+                        );
+                        expect(userFollowersSearchContainer).toBeTruthy();
+                    });
+                },
+            },
+            states: {
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏I find interesting that you nested states here. This is definitely an important feature of model-based testing.